distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -3073,7 +3073,7 @@ Portecle (http://portecle.sourceforge.net/) can be used for *keytool* operations
 The following steps define the procedure for creating a root CA to sign certificates.
 
 . Create a key pair. +
- `$> openssl genrsa -aes128 -out root-ca.key 1024`
+ `$> openssl genrsa -des3 -out root-ca.key 1024`
```

Comment:
We don't want to tell people to use 3DES, they should be using AES. Can you revert this part?